Wed, 09 Mar 2022 16:33:42-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：正常充值
Wed, 09 Mar 2022 16:33:42-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 100}
Wed, 09 Mar 2022 16:33:42-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'Success', 'msg': ''}
Wed, 09 Mar 2022 16:33:42-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'BadRequest', 'data': None, 'msg': '充值金额最低1元'}

Wed, 09 Mar 2022 16:33:42-_jb_pytest_runner.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'BadRequest' == 'Success'

Wed, 09 Mar 2022 16:33:42-_jb_pytest_runner.py-test_RechargeCase.py-[line:56]-ERROR-[日志信息]: 用例执行失败assert 'BadRequest' == 'Success'
Wed, 09 Mar 2022 16:33:43-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：不输入金额
Wed, 09 Mar 2022 16:33:43-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': None}
Wed, 09 Mar 2022 16:33:43-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:33:43-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:33:44-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：输入负数
Wed, 09 Mar 2022 16:33:44-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': -1}
Wed, 09 Mar 2022 16:33:44-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'WxClientError', 'msg': '生成微信订单失败, 请稍后重试'}
Wed, 09 Mar 2022 16:33:44-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'BadRequest', 'data': None, 'msg': '充值金额最低1元'}

Wed, 09 Mar 2022 16:33:44-_jb_pytest_runner.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'BadRequest' == 'WxClientError'

Wed, 09 Mar 2022 16:33:44-_jb_pytest_runner.py-test_RechargeCase.py-[line:56]-ERROR-[日志信息]: 用例执行失败assert 'BadRequest' == 'WxClientError'
Wed, 09 Mar 2022 16:33:44-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：输入字母
Wed, 09 Mar 2022 16:33:44-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 'adsadsa'}
Wed, 09 Mar 2022 16:33:44-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'WxClientError', 'msg': '生成微信订单失败, 请稍后重试'}
Wed, 09 Mar 2022 16:33:44-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:33:44-_jb_pytest_runner.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'RequestParameterIsWrong' == 'WxClientError'

Wed, 09 Mar 2022 16:33:44-_jb_pytest_runner.py-test_RechargeCase.py-[line:56]-ERROR-[日志信息]: 用例执行失败assert 'RequestParameterIsWrong' == 'WxClientError'
Wed, 09 Mar 2022 16:33:45-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：输入汉字
Wed, 09 Mar 2022 16:33:45-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': '测试测试'}
Wed, 09 Mar 2022 16:33:45-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'WxClientError', 'msg': '生成微信订单失败, 请稍后重试'}
Wed, 09 Mar 2022 16:33:45-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:33:45-_jb_pytest_runner.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'RequestParameterIsWrong' == 'WxClientError'

Wed, 09 Mar 2022 16:33:45-_jb_pytest_runner.py-test_RechargeCase.py-[line:56]-ERROR-[日志信息]: 用例执行失败assert 'RequestParameterIsWrong' == 'WxClientError'
Wed, 09 Mar 2022 16:33:47-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：充值大于100万
Wed, 09 Mar 2022 16:33:47-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 10000000000}
Wed, 09 Mar 2022 16:33:47-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'WxClientError', 'msg': '生成微信订单失败, 请稍后重试'}
Wed, 09 Mar 2022 16:33:47-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': {'code_url': 'https://qr.alipay.com/upx07895te9jsbnrtf2h0056', 'recharge_uuid': '013035959972127138', 'asset': 10000000000}, 'msg': ''}

Wed, 09 Mar 2022 16:33:47-_jb_pytest_runner.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'Success' == 'WxClientError'

Wed, 09 Mar 2022 16:33:47-_jb_pytest_runner.py-test_RechargeCase.py-[line:56]-ERROR-[日志信息]: 用例执行失败assert 'Success' == 'WxClientError'
Wed, 09 Mar 2022 16:33:47-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：充值少于1元
Wed, 09 Mar 2022 16:33:47-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 100}
Wed, 09 Mar 2022 16:33:47-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 16:33:47-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'BadRequest', 'data': None, 'msg': '充值金额最低1元'}

Wed, 09 Mar 2022 16:33:48-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：支付0元
Wed, 09 Mar 2022 16:33:48-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 0}
Wed, 09 Mar 2022 16:33:48-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 16:33:48-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:33:48-_jb_pytest_runner.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'RequestParameterIsWrong' == 'BadRequest'

Wed, 09 Mar 2022 16:33:48-_jb_pytest_runner.py-test_RechargeCase.py-[line:56]-ERROR-[日志信息]: 用例执行失败assert 'RequestParameterIsWrong' == 'BadRequest'
Wed, 09 Mar 2022 16:33:48-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：充值金额为空
Wed, 09 Mar 2022 16:33:48-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': None}
Wed, 09 Mar 2022 16:33:48-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 16:33:48-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:33:48-_jb_pytest_runner.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'RequestParameterIsWrong' == 'BadRequest'

Wed, 09 Mar 2022 16:33:48-_jb_pytest_runner.py-test_RechargeCase.py-[line:56]-ERROR-[日志信息]: 用例执行失败assert 'RequestParameterIsWrong' == 'BadRequest'
Wed, 09 Mar 2022 16:39:02-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：正常充值
Wed, 09 Mar 2022 16:39:02-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 1000}
Wed, 09 Mar 2022 16:39:02-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'Success', 'msg': ''}
Wed, 09 Mar 2022 16:39:02-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': {'code_url': 'weixin://wxpay/bizpayurl?pr=wyiYWNMzz', 'recharge_uuid': '708800400043945215', 'asset': 1000}, 'msg': ''}

Wed, 09 Mar 2022 16:39:03-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：不输入金额
Wed, 09 Mar 2022 16:39:03-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': None}
Wed, 09 Mar 2022 16:39:03-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:39:03-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:39:03-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：输入负数
Wed, 09 Mar 2022 16:39:03-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': -1}
Wed, 09 Mar 2022 16:39:03-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 16:39:03-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'BadRequest', 'data': None, 'msg': '充值金额最低1元'}

Wed, 09 Mar 2022 16:39:04-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：输入字母
Wed, 09 Mar 2022 16:39:04-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 'adsadsa'}
Wed, 09 Mar 2022 16:39:04-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:39:04-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:39:04-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：输入汉字
Wed, 09 Mar 2022 16:39:04-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': '测试测试'}
Wed, 09 Mar 2022 16:39:04-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:39:04-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:39:05-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：充值大于100万
Wed, 09 Mar 2022 16:39:05-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 10000000000}
Wed, 09 Mar 2022 16:39:05-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'WxClientError', 'msg': '生成微信订单失败, 请稍后重试'}
Wed, 09 Mar 2022 16:39:05-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'WxClientError', 'data': None, 'msg': '生成微信订单失败, 请稍后重试'}

Wed, 09 Mar 2022 16:39:05-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：充值少于1元
Wed, 09 Mar 2022 16:39:05-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 100}
Wed, 09 Mar 2022 16:39:05-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 16:39:05-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'BadRequest', 'data': None, 'msg': '充值金额最低1元'}

Wed, 09 Mar 2022 16:39:06-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：支付0元
Wed, 09 Mar 2022 16:39:06-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 0}
Wed, 09 Mar 2022 16:39:06-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:39:06-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:39:06-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：充值金额为空
Wed, 09 Mar 2022 16:39:06-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': None}
Wed, 09 Mar 2022 16:39:06-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:39:06-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:39:08-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：正常充值
Wed, 09 Mar 2022 16:39:08-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 1000}
Wed, 09 Mar 2022 16:39:08-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'Success', 'msg': ''}
Wed, 09 Mar 2022 16:39:08-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': {'code_url': 'https://qr.alipay.com/upx07392r241486bp4lp00bb', 'recharge_uuid': '230046824915925165', 'asset': 1000}, 'msg': ''}

Wed, 09 Mar 2022 16:39:09-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：不输入金额
Wed, 09 Mar 2022 16:39:09-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': None}
Wed, 09 Mar 2022 16:39:09-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:39:09-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:39:09-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：输入负数
Wed, 09 Mar 2022 16:39:09-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': -1}
Wed, 09 Mar 2022 16:39:09-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 16:39:09-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'BadRequest', 'data': None, 'msg': '充值金额最低1元'}

Wed, 09 Mar 2022 16:39:10-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：输入字母
Wed, 09 Mar 2022 16:39:10-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 'adsadsa'}
Wed, 09 Mar 2022 16:39:10-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:39:10-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:39:10-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：输入汉字
Wed, 09 Mar 2022 16:39:10-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': '测试测试'}
Wed, 09 Mar 2022 16:39:10-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:39:10-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:39:11-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：充值大于100万
Wed, 09 Mar 2022 16:39:11-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 10000000000}
Wed, 09 Mar 2022 16:39:11-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'WxClientError', 'msg': '生成微信订单失败, 请稍后重试'}
Wed, 09 Mar 2022 16:39:11-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': {'code_url': 'https://qr.alipay.com/upx06257byonfip5jfs330b0', 'recharge_uuid': '249256581246132239', 'asset': 10000000000}, 'msg': ''}

Wed, 09 Mar 2022 16:39:11-_jb_pytest_runner.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'Success' == 'WxClientError'

Wed, 09 Mar 2022 16:39:11-_jb_pytest_runner.py-test_RechargeCase.py-[line:56]-ERROR-[日志信息]: 用例执行失败assert 'Success' == 'WxClientError'
Wed, 09 Mar 2022 16:39:12-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：充值少于1元
Wed, 09 Mar 2022 16:39:12-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 100}
Wed, 09 Mar 2022 16:39:12-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 16:39:12-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'BadRequest', 'data': None, 'msg': '充值金额最低1元'}

Wed, 09 Mar 2022 16:39:12-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：支付0元
Wed, 09 Mar 2022 16:39:12-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 0}
Wed, 09 Mar 2022 16:39:12-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:39:12-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:39:13-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：充值金额为空
Wed, 09 Mar 2022 16:39:13-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': None}
Wed, 09 Mar 2022 16:39:13-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:39:13-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:52:19-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：正常充值
Wed, 09 Mar 2022 16:52:19-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 1000}
Wed, 09 Mar 2022 16:52:19-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'Success', 'msg': ''}
Wed, 09 Mar 2022 16:52:19-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': {'code_url': 'weixin://wxpay/bizpayurl?pr=WmRvSQ7zz', 'recharge_uuid': '092613052183172990', 'asset': 1000}, 'msg': ''}

Wed, 09 Mar 2022 16:52:19-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：不输入金额
Wed, 09 Mar 2022 16:52:19-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': None}
Wed, 09 Mar 2022 16:52:19-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:52:19-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:52:20-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：输入负数
Wed, 09 Mar 2022 16:52:20-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': -1}
Wed, 09 Mar 2022 16:52:20-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 16:52:20-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'BadRequest', 'data': None, 'msg': '充值金额最低1元'}

Wed, 09 Mar 2022 16:52:20-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：输入字母
Wed, 09 Mar 2022 16:52:20-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 'adsadsa'}
Wed, 09 Mar 2022 16:52:20-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:52:20-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:52:21-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：输入汉字
Wed, 09 Mar 2022 16:52:21-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': '测试测试'}
Wed, 09 Mar 2022 16:52:21-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:52:21-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:52:21-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：充值大于100万
Wed, 09 Mar 2022 16:52:21-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 10000000000}
Wed, 09 Mar 2022 16:52:21-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'WxClientError', 'msg': '生成微信订单失败, 请稍后重试'}
Wed, 09 Mar 2022 16:52:21-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'WxClientError', 'data': None, 'msg': '生成微信订单失败, 请稍后重试'}

Wed, 09 Mar 2022 16:52:22-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：充值少于1元
Wed, 09 Mar 2022 16:52:22-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 100}
Wed, 09 Mar 2022 16:52:22-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 16:52:22-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'BadRequest', 'data': None, 'msg': '充值金额最低1元'}

Wed, 09 Mar 2022 16:52:22-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：支付0元
Wed, 09 Mar 2022 16:52:22-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 0}
Wed, 09 Mar 2022 16:52:22-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:52:22-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:52:23-_jb_pytest_runner.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前用例名称：充值金额为空
Wed, 09 Mar 2022 16:52:23-_jb_pytest_runner.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': None}
Wed, 09 Mar 2022 16:52:23-_jb_pytest_runner.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:52:23-_jb_pytest_runner.py-test_RechargeCase.py-[line:34]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:52:25-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：正常充值
Wed, 09 Mar 2022 16:52:25-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 1000}
Wed, 09 Mar 2022 16:52:25-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'Success', 'msg': ''}
Wed, 09 Mar 2022 16:52:25-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': {'code_url': 'https://qr.alipay.com/upx01183vakxggqgtoy12563', 'recharge_uuid': '089804465399872723', 'asset': 1000}, 'msg': ''}

Wed, 09 Mar 2022 16:52:25-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：不输入金额
Wed, 09 Mar 2022 16:52:25-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': None}
Wed, 09 Mar 2022 16:52:25-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:52:25-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:52:26-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：输入负数
Wed, 09 Mar 2022 16:52:26-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': -1}
Wed, 09 Mar 2022 16:52:26-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 16:52:26-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'BadRequest', 'data': None, 'msg': '充值金额最低1元'}

Wed, 09 Mar 2022 16:52:26-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：输入字母
Wed, 09 Mar 2022 16:52:26-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 'adsadsa'}
Wed, 09 Mar 2022 16:52:26-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:52:26-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:52:27-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：输入汉字
Wed, 09 Mar 2022 16:52:27-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': '测试测试'}
Wed, 09 Mar 2022 16:52:27-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:52:27-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:52:27-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：充值少于1元
Wed, 09 Mar 2022 16:52:27-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 100}
Wed, 09 Mar 2022 16:52:27-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 16:52:27-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'BadRequest', 'data': None, 'msg': '充值金额最低1元'}

Wed, 09 Mar 2022 16:52:28-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：支付0元
Wed, 09 Mar 2022 16:52:28-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 0}
Wed, 09 Mar 2022 16:52:28-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:52:28-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 16:52:28-_jb_pytest_runner.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例名称：充值金额为空
Wed, 09 Mar 2022 16:52:28-_jb_pytest_runner.py-test_RechargeCase.py-[line:49]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': None}
Wed, 09 Mar 2022 16:52:28-_jb_pytest_runner.py-test_RechargeCase.py-[line:50]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 16:52:28-_jb_pytest_runner.py-test_RechargeCase.py-[line:51]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误'}

Wed, 09 Mar 2022 17:32:41-run_all_case.py-run_all_case.py-[line:19]-INFO-[日志信息]: ==================================2022-03-09 17:32:41===================================
Wed, 09 Mar 2022 17:32:42-run_all_case.py-test_InstanceCase.py-[line:36]-INFO-[日志信息]: 当前用例名称：创建实例-主机ID：484411a80c
Wed, 09 Mar 2022 17:32:42-run_all_case.py-test_InstanceCase.py-[line:37]-INFO-[日志信息]: 当前测试用例请求参数：{'instance_info': {'charge_type': 'payg', 'image': 'hub.kce.ksyun.com/autodl-image/torch:cuda11.0-cudnn8-devel-ubuntu18.04-py38-torch1.7.0', 'machine_id': '484411a80c', 'instance_name': '', 'req_gpu_amount': 1}, 'price_info': {'charge_type': 'payg', 'duration': 1, 'machine_id': '484411a80c', 'num': 1}}
Wed, 09 Mar 2022 17:32:42-run_all_case.py-test_InstanceCase.py-[line:38]-INFO-[日志信息]: 当前用例预期结果：{'code': 'Success', 'msg': ''}
Wed, 09 Mar 2022 17:32:42-run_all_case.py-test_InstanceCase.py-[line:39]-INFO-[日志信息]: 当前用例实际结果：{'code': 'InternalError', 'data': None, 'msg': '机器网络或存储异常，暂不可用，请更换机器', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:42-run_all_case.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'InternalError' == 'Success'
  - Success
  + InternalError

Wed, 09 Mar 2022 17:32:42-run_all_case.py-test_InstanceCase.py-[line:43]-ERROR-[日志信息]: 用例执行失败assert 'InternalError' == 'Success'
  - Success
  + InternalError
Wed, 09 Mar 2022 17:32:44-run_all_case.py-test_InstanceCase.py-[line:36]-INFO-[日志信息]: 当前用例名称：创建实例-主机ID：f94411a60c
Wed, 09 Mar 2022 17:32:44-run_all_case.py-test_InstanceCase.py-[line:37]-INFO-[日志信息]: 当前测试用例请求参数：{'instance_info': {'charge_type': 'payg', 'image': 'hub.kce.ksyun.com/autodl-image/torch:cuda11.0-cudnn8-devel-ubuntu18.04-py38-torch1.7.0', 'machine_id': 'f94411a60c', 'instance_name': '', 'req_gpu_amount': 1}, 'price_info': {'charge_type': 'payg', 'duration': 1, 'machine_id': 'f94411a60c', 'num': 1}}
Wed, 09 Mar 2022 17:32:44-run_all_case.py-test_InstanceCase.py-[line:38]-INFO-[日志信息]: 当前用例预期结果：{'code': 'Success', 'msg': ''}
Wed, 09 Mar 2022 17:32:44-run_all_case.py-test_InstanceCase.py-[line:39]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': 'f94411a60c-6e56f676', 'msg': ''}

Wed, 09 Mar 2022 17:32:45-run_all_case.py-test_InstanceCase.py-[line:36]-INFO-[日志信息]: 当前用例名称：创建实例-主机ID：484411a80c
Wed, 09 Mar 2022 17:32:45-run_all_case.py-test_InstanceCase.py-[line:37]-INFO-[日志信息]: 当前测试用例请求参数：{'instance_info': {'charge_type': 'payg', 'image': 'hub.kce.ksyun.com/autodl-image/torch:cuda11.0-cudnn8-devel-ubuntu18.04-py38-torch1.7.0', 'machine_id': '484411a80c', 'instance_name': '', 'req_gpu_amount': 1}, 'price_info': {'charge_type': 'payg', 'duration': 1, 'machine_id': '484411a80c', 'num': 1}}
Wed, 09 Mar 2022 17:32:45-run_all_case.py-test_InstanceCase.py-[line:38]-INFO-[日志信息]: 当前用例预期结果：{'code': 'Success', 'msg': ''}
Wed, 09 Mar 2022 17:32:45-run_all_case.py-test_InstanceCase.py-[line:39]-INFO-[日志信息]: 当前用例实际结果：{'code': 'InternalError', 'data': None, 'msg': '机器网络或存储异常，暂不可用，请更换机器', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:45-run_all_case.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'InternalError' == 'Success'
  - Success
  + InternalError

Wed, 09 Mar 2022 17:32:45-run_all_case.py-test_InstanceCase.py-[line:43]-ERROR-[日志信息]: 用例执行失败assert 'InternalError' == 'Success'
  - Success
  + InternalError
Wed, 09 Mar 2022 17:32:46-run_all_case.py-test_InstanceCase.py-[line:36]-INFO-[日志信息]: 当前用例名称：创建实例-主机ID：f94411a60c
Wed, 09 Mar 2022 17:32:46-run_all_case.py-test_InstanceCase.py-[line:37]-INFO-[日志信息]: 当前测试用例请求参数：{'instance_info': {'charge_type': 'payg', 'image': 'hub.kce.ksyun.com/autodl-image/torch:cuda11.0-cudnn8-devel-ubuntu18.04-py38-torch1.7.0', 'machine_id': 'f94411a60c', 'instance_name': '', 'req_gpu_amount': 1}, 'price_info': {'charge_type': 'payg', 'duration': 1, 'machine_id': 'f94411a60c', 'num': 1}}
Wed, 09 Mar 2022 17:32:46-run_all_case.py-test_InstanceCase.py-[line:38]-INFO-[日志信息]: 当前用例预期结果：{'code': 'Success', 'msg': ''}
Wed, 09 Mar 2022 17:32:46-run_all_case.py-test_InstanceCase.py-[line:39]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': 'f94411a60c-43802198', 'msg': ''}

Wed, 09 Mar 2022 17:32:46-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：手机号密码都正确
Wed, 09 Mar 2022 17:32:46-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': '18801053303', 'password': 'a753c776ff3ed4fefa2af948af87448910153281', 'captcha_id': '111'}
Wed, 09 Mar 2022 17:32:46-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'Success', 'msg': ''}
Wed, 09 Mar 2022 17:32:46-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': {'ticket': 'd95o2uzYI7q7tY7bHI4U1xBug7s=', 'user': {'id': 3, 'created_at': '2021-08-06T21:36:59+08:00', 'updated_at': '2021-09-14T10:40:09+08:00', 'uuid': '487686b5-637f-4d99-8202-7c0b29fa322d', 'username': 'zty_tester', 'nickname': 'ZhangTy', 'status': 'normal', 'logged_in_at': '2022-03-08T17:39:19.237+08:00', 'email': '', 'phone': '18801053303', 'is_admin': False, 'is_super_admin': False, 'sha1_password': 'a753c776ff3ed4fefa2af948af87448910153281', 'email_confirmed': False, 'idle_job_authority': True, 'mount_net_disk_authority': True, 'invitation_code': '', 'open_id': '111', 'union_id': '', 'applet_open_id': '', 'avatar_id': 0, 'register_ip': '', 'growth_value': -251000, 'member_level': 'normal_user', 'level_acq_mode': 'vip_user_acquire', 'member_dateline': None}}, 'msg': ''}

Wed, 09 Mar 2022 17:32:46-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：手机号正确，密码错误
Wed, 09 Mar 2022 17:32:46-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': '18801053303', 'password': '40bd001563085fc35165329ea1ff5c5ecbdbbeef'}
Wed, 09 Mar 2022 17:32:46-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:46-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': 1, 'msg': '手机号或密码错误，请重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：手机号错误，密码正确
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': '188010533', 'password': 'a753c776ff3ed4fefa2af948af87448910153281'}
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': 1, 'msg': '手机号或密码错误，请重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：手机号错误，密码错误
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': '18801053124', 'password': 'a8ef6091e7aded9675e79ff635ffd80e8c780420'}
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': 1, 'msg': '手机号或密码错误，请重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：手机号输入9位，密码输入正确
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': '188010500', 'password': 'a753c776ff3ed4fefa2af948af87448910153281'}
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': 1, 'msg': '手机号或密码错误，请重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：手机号输入12位，密码输入正确
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': '188010500001', 'password': 'a753c776ff3ed4fefa2af948af87448910153281'}
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': 1, 'msg': '手机号或密码错误，请重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：密码输入7位，手机号输入正确
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': '188010500001', 'password': '20eabe5d64b0e216796e834f52d61fd0b70332fc'}
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': 2, 'msg': '手机号或密码错误，请重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：密码输入17位，手机号输入正确
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': '188010500001', 'password': 'f6cff2998c2f0fd52fb0e9891d10ca3b0cb41fc1'}
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:47-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': 3, 'msg': '手机号或密码错误，请重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：密码输入17位，手机号输入正确
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': '188010500001', 'password': 'f6cff2998c2f0fd52fb0e9891d10ca3b0cb41fc1'}
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': 4, 'msg': '手机号或密码错误，请重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：手机号输入英文+汉字，密码输入正确
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': 'test登录', 'password': 'a8ef6091e7aded9675e79ff635ffd80e8c780420'}
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': 1, 'msg': '手机号或密码错误，请重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：不输入手机号，密码输入正确
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': '', 'password': 'a8ef6091e7aded9675e79ff635ffd80e8c780420'}
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：不输入密码，手机号输入正确
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': '18801053303', 'password': 'da39a3ee5e6b4b0d3255bfef95601890afd80709'}
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': 2, 'msg': '手机号或密码错误，请重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：手机号输入特殊符号
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': '！@#￥%……&*（)', 'password': 'da39a3ee5e6b4b0d3255bfef95601890afd80709'}
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': 1, 'msg': '手机号或密码错误，请重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：手机号输入null 密码不输入
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': 'Null', 'password': 'da39a3ee5e6b4b0d3255bfef95601890afd80709'}
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:48-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': 1, 'msg': '手机号或密码错误，请重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:49-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：密码输入null 手机号不输入
Wed, 09 Mar 2022 17:32:49-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': '', 'password': '109085beaaa80ac89858b283a64f7c75d7e5bb12'}
Wed, 09 Mar 2022 17:32:49-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:49-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:49-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：手机号输入null 密码输入null
Wed, 09 Mar 2022 17:32:49-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': '', 'password': '109085beaaa80ac89858b283a64f7c75d7e5bb12'}
Wed, 09 Mar 2022 17:32:49-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:49-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:49-run_all_case.py-test_LoginCase.py-[line:27]-INFO-[日志信息]: 当前用例名称：输入sql语句
Wed, 09 Mar 2022 17:32:49-run_all_case.py-test_LoginCase.py-[line:28]-INFO-[日志信息]: 当前测试用例请求参数：{'phone': 'select username，password from user where username=', 'password': 'e693f99fbb5efd10efe806601ff8bfebb64a4343'}
Wed, 09 Mar 2022 17:32:49-run_all_case.py-test_LoginCase.py-[line:29]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '手机号或密码错误，请重试'}
Wed, 09 Mar 2022 17:32:49-run_all_case.py-test_LoginCase.py-[line:30]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': 1, 'msg': '手机号或密码错误，请重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:50-run_all_case.py-test_RechargeCase.py-[line:30]-INFO-[日志信息]: 当前用例名称：正常充值
Wed, 09 Mar 2022 17:32:50-run_all_case.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 1000}
Wed, 09 Mar 2022 17:32:50-run_all_case.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前用例预期结果：{'code': 'Success', 'msg': ''}
Wed, 09 Mar 2022 17:32:50-run_all_case.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': {'code_url': 'weixin://wxpay/bizpayurl?pr=mNjRrqpzz', 'recharge_uuid': '107477879033115735', 'asset': 1000}, 'msg': ''}

Wed, 09 Mar 2022 17:32:50-run_all_case.py-test_RechargeCase.py-[line:30]-INFO-[日志信息]: 当前用例名称：不输入金额
Wed, 09 Mar 2022 17:32:50-run_all_case.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': None}
Wed, 09 Mar 2022 17:32:50-run_all_case.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 17:32:50-run_all_case.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:51-run_all_case.py-test_RechargeCase.py-[line:30]-INFO-[日志信息]: 当前用例名称：输入负数
Wed, 09 Mar 2022 17:32:51-run_all_case.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': -1}
Wed, 09 Mar 2022 17:32:51-run_all_case.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 17:32:51-run_all_case.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例实际结果：{'code': 'WxClientError', 'data': None, 'msg': '生成微信订单失败, 请稍后重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:51-run_all_case.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'WxClientError' == 'BadRequest'
  - BadRequest
  + WxClientError

Wed, 09 Mar 2022 17:32:51-run_all_case.py-test_RechargeCase.py-[line:37]-ERROR-[日志信息]: 用例执行失败assert 'WxClientError' == 'BadRequest'
  - BadRequest
  + WxClientError
Wed, 09 Mar 2022 17:32:51-run_all_case.py-test_RechargeCase.py-[line:30]-INFO-[日志信息]: 当前用例名称：输入字母
Wed, 09 Mar 2022 17:32:51-run_all_case.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 'adsadsa'}
Wed, 09 Mar 2022 17:32:51-run_all_case.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 17:32:51-run_all_case.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:52-run_all_case.py-test_RechargeCase.py-[line:30]-INFO-[日志信息]: 当前用例名称：输入汉字
Wed, 09 Mar 2022 17:32:52-run_all_case.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': '测试测试'}
Wed, 09 Mar 2022 17:32:52-run_all_case.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 17:32:52-run_all_case.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:52-run_all_case.py-test_RechargeCase.py-[line:30]-INFO-[日志信息]: 当前用例名称：充值大于100万
Wed, 09 Mar 2022 17:32:52-run_all_case.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 10000000000}
Wed, 09 Mar 2022 17:32:52-run_all_case.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前用例预期结果：{'code': 'WxClientError', 'msg': '生成微信订单失败, 请稍后重试'}
Wed, 09 Mar 2022 17:32:52-run_all_case.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例实际结果：{'code': 'WxClientError', 'data': None, 'msg': '生成微信订单失败, 请稍后重试', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:53-run_all_case.py-test_RechargeCase.py-[line:30]-INFO-[日志信息]: 当前用例名称：充值少于1元
Wed, 09 Mar 2022 17:32:53-run_all_case.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 100}
Wed, 09 Mar 2022 17:32:53-run_all_case.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 17:32:53-run_all_case.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': {'code_url': 'weixin://wxpay/bizpayurl?pr=ry2AgsQzz', 'recharge_uuid': '398678965654408691', 'asset': 100}, 'msg': ''}

Wed, 09 Mar 2022 17:32:53-run_all_case.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'Success' == 'BadRequest'
  - BadRequest
  + Success

Wed, 09 Mar 2022 17:32:53-run_all_case.py-test_RechargeCase.py-[line:37]-ERROR-[日志信息]: 用例执行失败assert 'Success' == 'BadRequest'
  - BadRequest
  + Success
Wed, 09 Mar 2022 17:32:54-run_all_case.py-test_RechargeCase.py-[line:30]-INFO-[日志信息]: 当前用例名称：支付0元
Wed, 09 Mar 2022 17:32:54-run_all_case.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 0}
Wed, 09 Mar 2022 17:32:54-run_all_case.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 17:32:54-run_all_case.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:54-run_all_case.py-test_RechargeCase.py-[line:30]-INFO-[日志信息]: 当前用例名称：充值金额为空
Wed, 09 Mar 2022 17:32:54-run_all_case.py-test_RechargeCase.py-[line:31]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': None}
Wed, 09 Mar 2022 17:32:54-run_all_case.py-test_RechargeCase.py-[line:32]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 17:32:54-run_all_case.py-test_RechargeCase.py-[line:33]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:56-run_all_case.py-test_RechargeCase.py-[line:45]-INFO-[日志信息]: 当前用例名称：正常充值
Wed, 09 Mar 2022 17:32:56-run_all_case.py-test_RechargeCase.py-[line:46]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 1000}
Wed, 09 Mar 2022 17:32:56-run_all_case.py-test_RechargeCase.py-[line:47]-INFO-[日志信息]: 当前用例预期结果：{'code': 'Success', 'msg': ''}
Wed, 09 Mar 2022 17:32:56-run_all_case.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': {'code_url': 'https://qr.alipay.com/upx00497h946plp6h8zv558e', 'recharge_uuid': '210670205734356763', 'asset': 1000}, 'msg': ''}

Wed, 09 Mar 2022 17:32:57-run_all_case.py-test_RechargeCase.py-[line:45]-INFO-[日志信息]: 当前用例名称：不输入金额
Wed, 09 Mar 2022 17:32:57-run_all_case.py-test_RechargeCase.py-[line:46]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': None}
Wed, 09 Mar 2022 17:32:57-run_all_case.py-test_RechargeCase.py-[line:47]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 17:32:57-run_all_case.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:58-run_all_case.py-test_RechargeCase.py-[line:45]-INFO-[日志信息]: 当前用例名称：输入负数
Wed, 09 Mar 2022 17:32:58-run_all_case.py-test_RechargeCase.py-[line:46]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': -1}
Wed, 09 Mar 2022 17:32:58-run_all_case.py-test_RechargeCase.py-[line:47]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 17:32:58-run_all_case.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': {'code_url': '', 'recharge_uuid': '112955585184471397', 'asset': -1}, 'msg': ''}

Wed, 09 Mar 2022 17:32:58-run_all_case.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'Success' == 'BadRequest'
  - BadRequest
  + Success

Wed, 09 Mar 2022 17:32:58-run_all_case.py-test_RechargeCase.py-[line:52]-ERROR-[日志信息]: 用例执行失败assert 'Success' == 'BadRequest'
  - BadRequest
  + Success
Wed, 09 Mar 2022 17:32:58-run_all_case.py-test_RechargeCase.py-[line:45]-INFO-[日志信息]: 当前用例名称：输入字母
Wed, 09 Mar 2022 17:32:58-run_all_case.py-test_RechargeCase.py-[line:46]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 'adsadsa'}
Wed, 09 Mar 2022 17:32:58-run_all_case.py-test_RechargeCase.py-[line:47]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 17:32:58-run_all_case.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:32:59-run_all_case.py-test_RechargeCase.py-[line:45]-INFO-[日志信息]: 当前用例名称：输入汉字
Wed, 09 Mar 2022 17:32:59-run_all_case.py-test_RechargeCase.py-[line:46]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': '测试测试'}
Wed, 09 Mar 2022 17:32:59-run_all_case.py-test_RechargeCase.py-[line:47]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 17:32:59-run_all_case.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:00-run_all_case.py-test_RechargeCase.py-[line:45]-INFO-[日志信息]: 当前用例名称：充值少于1元
Wed, 09 Mar 2022 17:33:00-run_all_case.py-test_RechargeCase.py-[line:46]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 100}
Wed, 09 Mar 2022 17:33:00-run_all_case.py-test_RechargeCase.py-[line:47]-INFO-[日志信息]: 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
Wed, 09 Mar 2022 17:33:00-run_all_case.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': {'code_url': 'https://qr.alipay.com/upx09891oqdfrtyjoxdr307b', 'recharge_uuid': '194181630007521868', 'asset': 100}, 'msg': ''}

Wed, 09 Mar 2022 17:33:00-run_all_case.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'Success' == 'BadRequest'
  - BadRequest
  + Success

Wed, 09 Mar 2022 17:33:00-run_all_case.py-test_RechargeCase.py-[line:52]-ERROR-[日志信息]: 用例执行失败assert 'Success' == 'BadRequest'
  - BadRequest
  + Success
Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RechargeCase.py-[line:45]-INFO-[日志信息]: 当前用例名称：支付0元
Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RechargeCase.py-[line:46]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': 0}
Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RechargeCase.py-[line:47]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RechargeCase.py-[line:45]-INFO-[日志信息]: 当前用例名称：充值金额为空
Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RechargeCase.py-[line:46]-INFO-[日志信息]: 当前测试用例请求参数：{'asset': None}
Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RechargeCase.py-[line:47]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RechargeCase.py-[line:48]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：正确注册用户
Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '123456aa', 'phone': '18349635870', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'Success', 'msg': ''}
Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'Success', 'data': {'ticket': 'KBKXar_lnMOwrxFmc7G8E7CyogE=', 'user': {'id': 1749, 'created_at': '2022-03-09T17:33:01+08:00', 'updated_at': '2022-03-09T17:33:01+08:00', 'uuid': 'c9571aa9-247d-4df6-980a-129e7ca525e4', 'username': '炼丹师5870', 'nickname': '炼丹师5870', 'status': 'normal', 'logged_in_at': None, 'email': '', 'phone': '18349635870', 'is_admin': False, 'is_super_admin': False, 'sha1_password': '123456aa', 'email_confirmed': False, 'idle_job_authority': False, 'mount_net_disk_authority': True, 'invitation_code': '', 'open_id': '', 'union_id': '', 'applet_open_id': '', 'avatar_id': 0, 'register_ip': '172.17.9.1', 'growth_value': 0, 'member_level': '', 'level_acq_mode': '', 'member_dateline': None}}, 'msg': ''}

Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：手机号输入10位
Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '123456aa', 'phone': '1880105330', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'InternalError', 'msg': '手机号格式错误'}
Wed, 09 Mar 2022 17:33:01-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'InternalError', 'data': None, 'msg': '手机号格式错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：手机号输入12位
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '123456aa', 'phone': '188010533033', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'InternalError', 'msg': '手机号格式错误'}
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'InternalError', 'data': None, 'msg': '手机号格式错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：手机号输入中文
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '123456aa', 'phone': '测试登录', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'InternalError', 'msg': '手机号格式错误'}
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'InternalError', 'data': None, 'msg': '手机号格式错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：手机号输入英文
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '123456aa', 'phone': 'qwertyuui', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'InternalError', 'msg': '手机号格式错误'}
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'InternalError', 'data': None, 'msg': '手机号格式错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：手机号输入特殊符号
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '123456aa', 'phone': '!@#$%^&*()', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'InternalError', 'msg': '手机号格式错误'}
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'InternalError', 'data': None, 'msg': '手机号格式错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：手机号输入11位数字+字母
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '123456aa', 'phone': '18801053303asdsadsa', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'InternalError', 'msg': '手机号格式错误'}
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'InternalError', 'data': None, 'msg': '手机号格式错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：手机号输入11位数字+汉字
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '123456aa', 'phone': '18801053303测试测试', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'InternalError', 'msg': '手机号格式错误'}
Wed, 09 Mar 2022 17:33:02-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'InternalError', 'data': None, 'msg': '手机号格式错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：手机号输入11位数字+特殊符号
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '123456aa', 'phone': '18801053303!@#$%', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'InternalError', 'msg': '手机号格式错误'}
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'InternalError', 'data': None, 'msg': '手机号格式错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：手机号为空
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '123456aa', 'phone': '', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：密码输入7位
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '1234567', 'phone': '18349635870', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'InternalError', 'msg': '验证码无效，请重新获取'}
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'AlreadyExist', 'data': None, 'msg': '手机号已注册', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:03-run_all_case.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'AlreadyExist' == 'InternalError'
  - InternalError
  + AlreadyExist

Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:36]-ERROR-[日志信息]: 用例执行失败assert 'AlreadyExist' == 'InternalError'
  - InternalError
  + AlreadyExist
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：密码输入17位
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '12345678901234567', 'phone': '18349635870', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'InternalError', 'msg': '验证码无效，请重新获取'}
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'AlreadyExist', 'data': None, 'msg': '手机号已注册', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:03-run_all_case.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'AlreadyExist' == 'InternalError'
  - InternalError
  + AlreadyExist

Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:36]-ERROR-[日志信息]: 用例执行失败assert 'AlreadyExist' == 'InternalError'
  - InternalError
  + AlreadyExist
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：密码只输入数字
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '12345678', 'phone': '18349635870', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'InternalError', 'msg': '验证码无效，请重新获取'}
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'AlreadyExist', 'data': None, 'msg': '手机号已注册', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:03-run_all_case.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'AlreadyExist' == 'InternalError'
  - InternalError
  + AlreadyExist

Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:36]-ERROR-[日志信息]: 用例执行失败assert 'AlreadyExist' == 'InternalError'
  - InternalError
  + AlreadyExist
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：密码只输入字母
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': 'asdfghjkl', 'phone': '18349635870', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'InternalError', 'msg': '验证码无效，请重新获取'}
Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'AlreadyExist', 'data': None, 'msg': '手机号已注册', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:03-run_all_case.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'AlreadyExist' == 'InternalError'
  - InternalError
  + AlreadyExist

Wed, 09 Mar 2022 17:33:03-run_all_case.py-test_RegisterCase.py-[line:36]-ERROR-[日志信息]: 用例执行失败assert 'AlreadyExist' == 'InternalError'
  - InternalError
  + AlreadyExist
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：密码输入数字+特殊符号
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': 'asdfghjkl', 'phone': '18349635870', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'InternalError', 'msg': '验证码无效，请重新获取'}
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'AlreadyExist', 'data': None, 'msg': '手机号已注册', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:04-run_all_case.py-Assert.py-[line:110]-ERROR-[日志信息]: Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'AlreadyExist' == 'InternalError'
  - InternalError
  + AlreadyExist

Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:36]-ERROR-[日志信息]: 用例执行失败assert 'AlreadyExist' == 'InternalError'
  - InternalError
  + AlreadyExist
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：密码为空
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '', 'phone': '18349635870', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：验证码输入错误，手机号、密码输入正确
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '123456aa', 'phone': '18843211234', 'v_code': '123456'}
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'InternalError', 'msg': '验证码无效，请重新获取'}
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'InternalError', 'data': None, 'msg': '验证码无效，请重新获取', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：验证码为空
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '123456aa', 'phone': '18349635870', 'v_code': ''}
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'RequestParameterIsWrong', 'msg': '请求参数错误'}
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'RequestParameterIsWrong', 'data': None, 'msg': '请求参数错误', 'debug_error': ''}

Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:28]-INFO-[日志信息]: 当前用例名称：注册过的手机号
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:29]-INFO-[日志信息]: 当前测试用例请求参数：{'password': '123456', 'phone': '18801053303', 'v_code': '666666'}
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:30]-INFO-[日志信息]: 当前用例预期结果：{'code': 'AlreadyExist', 'msg': '手机号已注册'}
Wed, 09 Mar 2022 17:33:04-run_all_case.py-test_RegisterCase.py-[line:31]-INFO-[日志信息]: 当前用例实际结果：{'code': 'AlreadyExist', 'data': None, 'msg': '手机号已注册', 'debug_error': ''}

